streetcrm/admin.py

```diff
@@ -547,13 +387,10 @@ def advanced_search_do(self, request, form, export):
             }
         else:
             results_package = {"form": form,
-                               "search_results": results,
-                               "event_results": event_list
+                               "search_results": results
```

Comment:
So all kinds of results will show up in `results`?  Thank you!!

Reply:
Yep! This just consolidates the results into one.